qt/python/mantidqt/widgets/common/observing_presenter.py

```diff
@@ -0,0 +1,47 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+
+
+class ObservingPresenter:
```

Comment:
Can we use a new-style class while we're still in Python 2?

Reply:
I've been running this on Python 2.7 and have had no issues